text/game/backstory.stringtable

```diff
@@ -631,12 +631,12 @@
     </Entry>
     <Entry>
       <ID>126</ID>
-      <DefaultText>Als Bestrafung für seine Verbrechen und seine persönlichen Beleidigungen deiner Person risst du Thaos' Seele in Stücke und vernichtetest ihn dadurch für immer.</DefaultText>
+      <DefaultText>Als Bestrafung für seine Verbrechen und seine persönlichen Angriffe gegen hast du Thaos' Seele in Stücke gerissen und ihn dadurch für immer vernichtet.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>127</ID>
-      <DefaultText>Durch Thaos' Tod warst du in der Lage, endlich eine Bestätigung des Ursprungs und der Natur der Götter zu erlangen, und diese Antworte scheint deiner Seele etwas Erleichterung zu bringen. Die Halluzinationen und das Geflüster gingen zurück. Nun blieb nur noch übrig, über die Zukunft der in Sonne im Schatten gefangenen Seelen zu entscheiden.</DefaultText>
+      <DefaultText>Durch Thaos' Tod hast du letztendlich eine Bestätigung über die Herkunft und die Natur der Götter erhalten, und diese Antwort schien deine Seele zu erleichtern. Die Halluzinationen und Stimmen verschwanden. Es bleibt nur noch zu entscheiden, was mit den gefangenen Seelen in Sonne im Schatten geschehen soll.</DefaultText>
```

Comment:
With Thaos’ death, you were able to receive confirmation at last of the origin and nature of the gods, and that answer seemed to bring some relief to your soul.